gradle-plugin/src/main/kotlin/mozilla/appservices/AppServicesExtension.kt

```diff
@@ -26,14 +26,16 @@ open class AppServicesExtension(project: Project) {
                 DefaultModuleIdentifier.newId("org.mozilla.appservices", "lockbox-megazord"),
                 setOf(
                         DefaultModuleIdentifier.newId("org.mozilla.fxaclient", "fxaclient"),
-                        DefaultModuleIdentifier.newId("org.mozilla.sync15", "logins")
+                        DefaultModuleIdentifier.newId("org.mozilla.sync15", "logins"),
+                        DefaultModuleIdentifier.newId("mozilla.appservices.rustlog", "rustlog")
```

Comment:
This doesn't seem quite right.  We want the Gradle coordinates to be "org.mozilla.appservices:rustlog", parallel to "org.mozilla.appservices:lockbox-megazord".  Let me find where this needs to change.